.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: OpenRA Forum
+    url: https://forum.openra.net/
+    about: "Please ask questions about modding here."
+  - name: OpenRA IRC Channel
```

Comment:
Ss much as I like Discord as a service, it offers a much better experience for new users. We should link to that instead of IRC IMO.

Reply:
Just fyi, the (double) quotes were only needed so that the parser does not treat `#openra` as a comment...